src/MessageBox/MarketerialLayout/MessageBoxMarketerialLayout.spec.js

```diff
@@ -67,6 +68,32 @@ describe('MessageBoxMarketerialLayout', () => {
         expect(await driver.getPrimaryButton()).toBeNull();
       });
 
+      it('should display primary button node if passed', async () => {
+        const props = Object.assign(
+          {},
+          requiredProps,
+          {
+            primaryButtonNode: <Button>Hi</Button>,
+          },
+          {},
+        );
+        const driver = createDriver(<MessageBoxMarketerialLayout {...props} />);
+        expect(await driver.getPrimaryButtonNode()).toBeTruthy();
```

Comment:
Also minor, but we prefer:
```suggestion
        expect(await driver.getPrimaryButtonNode()).toBe(true);
```

Because of:
https://vincenttunru.com/toBeTruthy-vs-toBe-true/